packages/nextra-theme-docs/src/components/anchor.tsx

```diff
@@ -41,7 +41,7 @@ export const Anchor = forwardRef<HTMLAnchorElement, AnchorProps>(function (
     )
   }
 
-  if (nextVersion > 12 || config.newNextLinkBehavior) {
+  if (nextVersion > 12 || config.frontMatter.newNextLinkBehavior) {
```

Comment:
this is not in frontMatter https://github.com/shuding/nextra/blob/1e26811bdd4f0e6ab571f93497ef116c06fee06c/packages/nextra-theme-docs/src/contexts/config.tsx#L108-L110, we can remove it in v3